action.yml

```diff
@@ -1,20 +1,23 @@
-name: Validate Tables
-description: Validates the presence of tables in the Deephaven server
+name: Assert Tables
+description: Asserts the presence of tables in a Deephaven server
 inputs:
   table-names:
     description: A string containing a comma separated list of table names to check
     required: true
   host:
-    description: The host name of the Deephaven instance
-    required: false
-    default: envoy
-  project-name:
-    description: The docker-compose project name to bundle separate docker-compose files
+    description: The host name or IP address of the Deephaven instance
+    required: true
+  docker-compose-project-name:
+    description: The docker-compose project name for the assertion
+    required: true
+  max-retries:
+    description: The maximum attempts to retry connecting to Deephaven
     required: true
```

Comment:
I would just give `max-retries` a default.

Reply:
Yeah that's reasonable, I'll make it 5